auditbeat/module/auditd/audit_linux.go

```diff
@@ -533,6 +537,50 @@ func buildMetricbeatEvent(msgs []*auparse.AuditMessage, config Config) mb.Event
 	return out
 }
 
+func normalizeEventFields(m common.MapStr) {
+	for _, condition := range []struct {
+		fields common.MapStr
+		action common.MapStr
+	}{
+		{
+			fields: common.MapStr{
```

Comment:
Would these maps be continuously allocated for each event it processes?

Reply:
Yes, I improved it, thanks